### PR TITLE
Story two

### PR DESCRIPTION
**Pull request by eemakim**
- Created modal displaying the job details of each job
- Job details pull through from API based on URL Job ID

### CODE REVIEW

File: src/components/Modal/index.jsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                  </div>
+                  <div className='flex flex-col gap-1'>
+                    <div>Skills</div>
+                      <div className='flex flex-row gap-1'>
+                        {jobSkills.map((skill) => (
+                        <span key={skill.id} className='bg-blue-500 text-white p-1 rounded-lg'>
+                          {skill.skill}
+                        </span>
+                      ))}
+                      </div>
```
Comment (ashleycoles) [inline src/components/Modal/index.jsx]: There's a small isssue here with responsivity, on a very small screen the skills get cut off

File: src/components/Modal/index.jsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+      setType(data.type != null ? data.type : 'N/A')
+      setImg(data.logo)
+      setDesc(data.job_description)
+      setSalary(data.salary != null ? data.salary : ' -')
+      setDate(data.posted)
+      setSkills(data.skills)
+    })
+  }
+  // console.log(jobTitle)
+  // left-1/2 top-1/2 transform -translate-x-1/2 -translate-y-1/2
```
Comment (ashleycoles) [inline src/components/Modal/index.jsx]: You should remove these comments

File: src/components/JobCard/index.jsx [hunk 3]
```diff
@@ -1,6 +1,13 @@
-function JobCard({ salary, type, title, company, logo, skills }) {
-  return (
-    <div className='flex flex-col even:bg-zinc-700 odd:bg-zinc-500' style={{ width: '100%' }}>
+import { useContext, useState } from "react";
+import modalContext from "../../context/modalContext";
+import JobModal from "../Modal";
```
Comment (ashleycoles) [inline src/components/JobCard/index.jsx]: Does this import need to be here?

**Review by ashleycoles — CHANGES_REQUESTED** [review]
Great work, a few comments for you!

**Comment by objen** [comment]
Should be fixed now @ashleycoles

**Review by ashleycoles — APPROVED** [review]

**Pull request merged**